src/components/application_manager/src/commands/mobile/alert_maneuver_request.cc

```diff
@@ -238,6 +238,25 @@ bool AlertManeuverRequest::IsWhiteSpaceExist() {
       }
     }
   }
+  if ((*message_)[strings::msg_params].keyExists(strings::soft_buttons)) {
+    const smart_objects::SmartArray* sb_array =
+        (*message_)[strings::msg_params][strings::soft_buttons].asArray();
+
+    smart_objects::SmartArray::const_iterator it_sb = sb_array->begin();
+    smart_objects::SmartArray::const_iterator it_sb_end = sb_array->end();
+
+    for (; it_sb != it_sb_end; ++it_sb) {
+      str = (*it_sb)[strings::text].asCharArray();
+      if (strlen(str) && !CheckSyntax(str)) {
+        LOG4CXX_ERROR(logger_, "Invalid soft_buttons syntax check failed");
+        return true;
+      } else if (!CheckSyntax(str, false)) {
+        LOG4CXX_ERROR(logger_, "Invalid soft_buttons syntax check failed");
```

Comment:
@okozlovlux Confusd logic , please make it more clear, at lease pass 'false' as variable with name. And log not the same errors,  and split it ti 3 if's with returns